scripts/bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+
+# Script to install Nix/Devbox/Direnv
+#
+#
+export DEVBOX_USE_VERSION="0.7.1"
+
+export NIX_SSL_CERT_FILE="$HOME/.local/state/system-certs.pem"
+
+save_netskope_cert() {
+	if [ ! -e "$NIX_SSL_CERT_FILE" ]; then
+		mkdir -p ~/.local/state/
+		security find-certificate -a -p /System/Library/Keychains/SystemRootCertificates.keychain /Library/Keychains/System.keychain >"$NIX_SSL_CERT_FILE"
+	fi
+}
+
+install_nix() {
+	curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install --extra-conf "trusted-users = root $(whoami)" --ssl-cert-file "$NIX_SSL_CERT_FILE"
+
+}
+
+install_devbox() {
+	curl -fsSL https://get.jetpack.io/devbox | FORCE=1 bash
+}
+
+install_direnv() {
+	curl -fsSL https://get.jetpack.io/devbox | bin_path=/usr/local/bin bash
```

Comment:
does setting `bin_path` install devbox to a non-standard location? I'd prefer to keep things standard if we can

Reply:
When you don't specify `bin_path`, direnv install script is going to install it on the `first path it found in your $PATH env` 

It only have standard path if it's installed by homebrew/asdf/etc.

Reply:
but you're installing devbox not direnv?